packages/table/src/table.tsx

```diff
@@ -489,15 +489,34 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
-    public resizeRowsByTallestCell(columnIndex: number) {
+    /**
+     * Resize rows based on a given set of column indices.
+     * If no indices are provided, default to using the columns that are currently in the viewport.
+     */
+    public resizeRowsByTallestCells(columnIndices?: number[]) {
```

Comment:
Another thought: I don't think this function is named quite correctly as is; no matter how many column indices you provide, you still only care about one "tallest cell" amongst all of them, right?

Is `resizeRowsByTallestCell(columnIndices: number | number[])` out of the question? Then we could just follow a different code path if `Arrays.isArray(columnIndices) === true`.